tests/client/amo/components/TestErrorPage.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {
+  renderIntoDocument,
+  findRenderedComponentWithType,
+} from 'react-addons-test-utils';
+import { findDOMNode } from 'react-dom';
+import { Provider } from 'react-redux';
+
+import ErrorPage, { mapStateToProps } from 'amo/components/ErrorPage';
+import createStore from 'amo/store';
+import { REDUX_CONNECT_LOAD_FAIL } from 'core/constants';
+import { getFakeI18nInst } from 'tests/client/helpers';
+import I18nProvider from 'core/i18n/Provider';
+import { signedInApiState } from 'tests/client/amo/helpers';
+
+
+describe('<ErrorPage />', () => {
+  function render({ ...props }, store = createStore(signedInApiState)) {
+    return findDOMNode(findRenderedComponentWithType(renderIntoDocument(
+      <Provider store={store}>
+        <I18nProvider i18n={getFakeI18nInst()}>
+          <ErrorPage {...props} />
+        </I18nProvider>
+      </Provider>
+    ), ErrorPage));
+  }
+
+  it('renders children when there are no errors', () => {
+    const rootNode = render({ children: <div>hello</div> });
+
+    assert.equal(rootNode.textContent, 'hello');
+  });
+
+  it('renders an error page on error', () => {
+    const store = createStore(signedInApiState);
+    const payload = {
+      error: {
+        response: {
+          otherData: 'something',
+          status: 404,
+        },
+      },
+    };
+    store.dispatch({ type: REDUX_CONNECT_LOAD_FAIL, payload });
```

Comment:
None of the tests should dispatch this directly. They need to use the reduc-connect method:
````js
import { loadFail } from 'redux-connect/lib/store';
store.dispatch(loadFail('SomeKey', apiError));
````
This is because we need the test to ensure that our mimicry of redux-connect doesn't break.